### PR TITLE
Disable keybinder in wayland

### PR DESCRIPTION
**Pull request by JasonLG1979**
Effectively rewrote the mediakeys plugin to make it fail gracefully and make it easier to add support for more DE's. Fixes: https://github.com/pithos/pithos/issues/448

### CODE REVIEW

File: pithos/plugins/mediakeys.py [hunk 1]
```diff
@@ -25,125 +25,160 @@ class MediaKeyPlugin(PithosPlugin):
...
+            path = '/org/gnome/SettingsDaemon/MediaKeys'
+            bus_names = [
+                'org.gnome.SettingsDaemon',
+                'org.gnome.SettingDaemon.MediaKeys',
+            ]
+        elif mate:
+            name = 'org.mate.SettingsDaemon.MediaKeys'
+            path = '/org/mate/SettingsDaemon/MediaKeys'
+            # Presumably mate will follow gnome
+            # in changing the bus name at some point if they haven't already?
```
Comment (TingPing) [inline pithos/plugins/mediakeys.py]: I doubt it please verify these.

Comment (JasonLG1979) [inline pithos/plugins/mediakeys.py]: Ok, I'll remove them for now I just figured better safe than sorry.

Comment (JasonLG1979) [inline pithos/plugins/mediakeys.py]: Done.

**Comment by JasonLG1979** [comment]
@TingPing can we get this and the add stations in the treeview PR in and release 1.3.1 please?

**Pull request merged**